app/views/add-project.html

```diff
@@ -0,0 +1,94 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+	Add Project
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+
+	<div class="grid-row">
+		<section style="padding-top:30px;">
+			<form action="/api/projects/add" method="post" class="form" id="addProjectForm">
+				
+				<h1 class="form-title heading-large">
+					Add Project
+				</h1>
+			
+				<!-- Project name (title) -->
+				<div class="form-group">
+					<label class="form-label-bold" for="name">Title</label>
+					<p class="form-hint">Name of the project</p>
+					<input type="text" class="form-control" id="name" name="name">
+				</div>
+			
+				<!-- Location -->
+				<div class="form-group">
+					<label class="form-label-bold" for="location">
+						Location
+						<span class="form-hint">
+							Location of the team working on the project, e.g. London
+						</span>
+					</label>
+					<input type="text" class="form-control" id="location" name="location">
+				</div>
+			
+				<!-- Description -->
+				<div class="form-group">
+					<label class="form-label-bold" for="description">
+						Description
+						<span class="form-hint">
+							Description of the project
+						</span>
+					</label>
+					<textarea class="form-control" id="description" name="description"></textarea>
+				</div>
+
+				<!-- Phase -->
+				<div class="form-group">
+					<label class="form-label-bold" for="phase">
+						Phase
+						<span class="form-hint">
+							Phase of the project
+						</span>
+					</label>
+					<select class="form-control" name="phase" id="phase">
+						<option value="pipeline">Pipeline</option>
+						<option value="discovery">Discovery</option>
+						<option value="alpha">Alpha</option>
+						<option value="beta">Beta</option>
+						<option value="live">Live</option>
+					</select>
+				</div>
+			
+				<div class="form-group">
+					<input type="submit" class="button" value="Continue">
+				</div>
+			</form>
+		</section>
+	</div>
+</main>
+{% endblock %}
+
+{% block body_end %}
+{{ super() }}
+<script>
+$("#addProjectForm").submit(function(e) {
+    e.preventDefault();
+    $.ajax({
+        type: "POST",
+        url: $(this).attr('action'),
+        contentType: "application/json",
+        data: JSON.stringify($('#addProjectForm').serializeObject())
+
+    })
+    .done(function(response) {
+        alert(JSON.stringify(response.messages[0])); 
+    })
+    .fail(function(response) {
+        alert('There was an error adding project. ' + response.messages[0]);
```

Comment:
I think `.fail` returns `jqXHR` as the first parameter of the callback rather than the data, this will fail